packages/core/src/dispatcher.ts

```diff
@@ -47,19 +47,17 @@ const createActionDispatcher = <
 
 /**
  * Creates a dispatcher object for a model - it contains a mapping from all
- * reducers and effects *names* to functions which dispatch their corresponding
- * actions.
+ * reducers to functions which dispatch their corresponding actions.
  */
-const createDispatcher = <
+export const createDispatcher = <
```

Comment:
Maybe calling it `createReducerDispatcher` is better?

Reply:
Done!